### PR TITLE
Configurable fastd connection attempts

### LINKED ISSUE #133: Configure connection attempts by unknown peers
after this line:

[`include peers from "peers"`](https://github.com/ffnord/ffnord-puppet-gateway/blob/master/templates/etc/fastd/fastd.conf.erb#L14)

 we could add:

    # Configure connection attempts by unknown peers
    # "true" means, all attempts are accepted or a shell command that is run on attempts
    on verify "true";
    #on verify "/bin/bash /path/to/fastd/verify.sh $PEER_KEY";

Optionally configure this in your gateway.pp

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

**Comment by rubo77** [comment]
This lets you define the content of 

    verify on

in the `fastd.conf` file.  default is an empty string.
if empty  the line will not be added at all to the config file so the default behaviour with the peers folder stays acive.

File: manifests/fastd.pp [hunk 1]
```diff
@@ -8,6 +8,7 @@
                      , $fastd_port
 
                      , $fastd_peers_git
+                     , $fastd_verify = ''
```
Comment (sargon) [inline manifests/fastd.pp]: Why not use a Boolean instead of an Integer that is used like a Boolean.

Comment (rubo77) [inline manifests/fastd.pp]: Because this could be empty, contain 'true' or a path to a script that verifies the connection attempt

**Comment by sargon** [comment]
You have tested this with stated values, have you?

**Comment by rubo77** [comment]
I tested with "true" and with "", but not with the script like suggested here https://github.com/ffnord/ffnord-puppet-gateway/issues/133

**Comment by sargon** [comment]
Try true, without the double-quotes.

**Comment by rubo77** [comment]
i don't know if it works without out the quotes. Must be with ! Maybe add this in a comment?

**Comment by rubo77** [comment]
Maybe we should rename the variable to

    $fastd_validate_override_value

But that is quite long.

**Comment by rubo77** [comment]
I added it in the comment

**Pull request merged**